### PR TITLE
Test for GCCompletedRedelegations

### PR DESCRIPTION
**Pull request by phamminh0811**
## 1. Summary
Fixes #617 

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

File: x/interchainstaking/keeper/redelegation_record_test.go [hunk 1]
```diff
@@ -51,3 +51,57 @@ func (suite *KeeperTestSuite) TestRedelegationRecordSetGetIterate() {
...
+
+	records = quicksilver.InterchainstakingKeeper.AllRedelegationRecords(ctx)
+	suite.Equal(3, len(records))
+
+	err := quicksilver.InterchainstakingKeeper.GCCompletedRedelegations(ctx)
+	suite.NoError(err)
+
+	records = quicksilver.InterchainstakingKeeper.AllRedelegationRecords(ctx)
+	suite.Equal(2, len(records))
+}
```
Comment (ThanhNhann) [inline x/interchainstaking/keeper/redelegation_record_test.go:107]: Can you add one more check `GetRedelegationRecord` with the deleted `RedelegationRecord` to verify after using `GCCompletedRedelegations`? I think all the code is good and we can merge after adding the check I mentioned, thank @phamminh0811

Comment (phamminh0811) [reply x/interchainstaking/keeper/redelegation_record_test.go:107]: Got it

**Review by ThanhNhann — APPROVED** [review]
lgtm!

**Comment by phamminh0811** [comment]
@joe-bowman could you check this pr for me?

**Review by joe-bowman — APPROVED** [review]
lgtm

**Pull request merged**